package.json

```diff
@@ -133,15 +133,11 @@
     "validator": "^3.39.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
     "@commitlint/cli": "^7.1.2",
     "@commitlint/config-angular": "^7.1.2",
     "@ovh-ux/manager-webpack-config": "^3.0.0",
     "angular-template-url-loader": "^1.0.2",
     "babel-eslint": "^8.2.6",
```

Comment:
Is this also needed then ?

Reply:
Yes it is. You can refer to the ESLint configuration file: https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/.eslintrc.json#L4 :)